### PR TITLE
508 missing impelementation type

### LINKED ISSUE #508: Implementation is missing on message throw and end events
__Describe the Bug__

> The message end event is missing an implementation. (...) Same for intermediate message throw event

__Steps to Reproduce__

1. Select a message end or throw event
2. now this happens

__Expected Behavior__

Implementation is available.

__Environment__

- Host (Browser/Node version), if applicable: [e.g. MS Edge 18, Chrome 69, Node 10 LTS]
- OS: [e.g. Windows 7]
- Library version: [e.g. 2.0.0]

### PR DESCRIPTION
**Pull request by MaxTru**
closes #508

### CODE REVIEW

File: src/provider/camunda-platform/properties/ImplementationProps.js [hunk 1]
```diff
@@ -23,11 +22,11 @@ import {
 
 
 export function ImplementationProps(props) {
-  const {
+  let {
     element
   } = props;
 
-  if (!is(element, 'camunda:ServiceTaskLike')) {
+  if (!getServiceTaskLikeBusinessObject(element)) {
```
Comment (pinussilvestrus) [inline src/provider/camunda-platform/properties/ImplementationProps.js:29]: Nice 👍

**Review by pinussilvestrus — APPROVED** [review]
Works 👍

File: src/provider/camunda-platform/properties/ImplementationProps.js [hunk 2]
```diff
@@ -23,11 +22,11 @@ import {
 
 
 export function ImplementationProps(props) {
-  const {
+  let {
```
Comment (barmac) [inline src/provider/camunda-platform/properties/ImplementationProps.js:25]: I can't see this overwritten anywhere. I believe we can leave it as `const`

Comment (barmac) [reply src/provider/camunda-platform/properties/ImplementationProps.js:25]: I haven't noticed it was already merged 🙈

Comment (MaxTru) [reply src/provider/camunda-platform/properties/ImplementationProps.js:25]: NVM: https://github.com/bpmn-io/bpmn-js-properties-panel/pull/519

**Pull request merged**